app/create_buyer/forms/auth_forms.py

```diff
@@ -1,14 +1,14 @@
 from flask_wtf import FlaskForm
-from wtforms.validators import DataRequired, Regexp
-from dmutils.forms import StripWhitespaceStringField
+from wtforms.validators import DataRequired
+from dmutils.forms.fields import DMStripWhitespaceStringField
+from dmutils.forms.validators import EmailValidator as ValidEmailAddress
 
 
 class EmailAddressForm(FlaskForm):
-    email_address = StripWhitespaceStringField(
-        'Email address', id="input_email_address",
+    email_address = DMStripWhitespaceStringField(
+        "Your email address",
         validators=[
             DataRequired(message="You must provide an email address"),
-            Regexp("^[^@^\s]+@[^@^\.^\s]+(\.[^@^\.^\s]+)+$",
-                   message="You must provide a valid email address")
+            ValidEmailAddress(message="You must provide a valid email address"),
```

Comment:
Surprised I missed this when checking for email validation doing my "use notify's validator" ticket. Think I was probably grepping for `ValidEmailAddress`, not expecting things to be bypassing it entirely.